models/game.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(sequelize, DataTypes) {
+  const Game = sequelize.define('Game', {
+    userName: DataTypes.STRING,
+    password: DataTypes.STRING,
+  });
```

Comment:
is there a reason there's two user objects?  I assume this is just remnants from a copy paste that will be added to/removed later.